### ISSUE TITLE
Mod rollback, or version selection.

### ISSUE THREAD

**Issue by gstrange42** [brianmiller/phvalheim-server#63]
I was thinking a good feature to add in the future is the ability to either roll back a mod update to the last version.  Or an option to select a previous version of the mod. 

I've nearly run into a situation where a mod update crippled the server, but the previous version was fine. I was lucky,  but it got me thinking of the cache keeps copies of previous versions would it be possible to select one of those and have it replace the current one. 

With options to never update, only update when a new version is released essentially skipping the current release.

**Comment by brianmiller** [comment]
Yeah, it's something we've talked about. It gets complicated keeping track of the different versions.
